packit_service/worker/helpers/build/copr_build.py

```diff
@@ -569,6 +574,26 @@ def handle_build_submit_error(self, ex):
         """
         Handle errors when submitting Copr build.
         """
+        if (
+            # don't retry the task if user action needed
+            not (
+                MISSING_PERMISSIONS_TO_BUILD_IN_COPR in str(ex)
+                or NOT_ALLOWED_TO_BUILD_IN_COPR in str(ex)
+            )
+            and not self.celery_task.is_last_try()
+        ):
+            interval = 1
```

Comment:
What about some backoff? In case of an outage, increasing the interval might be useful to not overwhelm Copr and ourselves.